actionx/ACTIONX_WELPI.DATA

```diff
@@ -305,10 +305,6 @@ SUMMARY
 --
 --       ACTIVATE COLUMNAR SUMMARY DATA REPORTING OPTION
 --
-RUNSUM
---
---       ACTIVATE SUMMARY DATA RSM FILE OUTPUT OPTION
---
```

Comment:
Not an expert here. Why is this removed?

Reply:
[Bard said](https://github.com/OPM/opm-tests/pull/1157#discussion_r1596601980) this summary keyword has created problems sometimes in the past when comparing two runs of flow and he suggested to remove it.

Reply:
Yep, just noticed that myself while looking at the history.